### PR TITLE
EVG-3234 Validate createhost command

### PR DESCRIPTION
**Pull request by bsamek**

### CODE REVIEW

File: model/generate.go [hunk 1]
```diff
@@ -16,6 +16,7 @@ const (
 	maxGeneratedBuildVariants = 100
 	maxGeneratedTasks         = 1000
 	GenerateTasksCommandName  = "generate.tasks"
+	CreateHostCommandName     = "createhost"
```
Comment (tychoish) [inline model/generate.go]: should we put these in globals? and have a global slice for the group?

Comment (john-m-liu) [inline model/generate.go]: We chose different names for the command :( do we want createhost or create.host?

Comment (bsamek) [reply model/generate.go]: done

Comment (bsamek) [reply model/generate.go]: done

**Pull request merged**